lib/kb_SetUtilities/kb_SetUtilitiesImpl.py

```diff
@@ -762,17 +725,17 @@ def KButil_Slice_FeatureSets_by_Genomes(self, ctx, params):
 
             # Build sliced FeatureSet
             #
-            self.log (console, "BUILDING SLICED FEATURESET\n")  # DEBUG
-            self.log (console, "Slicing out genomes "+("\n".join(params['input_genome_refs'])))  # DEBUG
+            self.log (console, "BUILDING SLICED FEATURESET\n")
+            self.log (console, "Slicing out genomes "+("\n".join(params['input_genome_refs'])))
             element_ordering = []
             elements = {}
             for fId in this_element_ordering:
-                #self.log (console, 'checking feature {}'.format(fId))  # DEBUG
+                #self.log (console, 'checking feature {}'.format(fId))
```

Comment:
can this either be uncommented or deleted?

Reply:
done